captum/attr/_models/base.py

```diff
@@ -21,20 +21,34 @@ class InterpretableEmbeddingBase(Embedding):
     """
 
     def __init__(self, embedding, full_name):
-        super().__init__(embedding.num_embeddings, embedding.embedding_dim)
+        Module.__init__(self)
+        self.num_embeddings = (
+            embedding.num_embeddings if hasattr(embedding, "num_embeddings") else None
+        )
+        self.embedding_dim = (
+            embedding.embedding_dim if hasattr(embedding, "embedding_dim") else None
```

Comment:
Can you just use `getattr` for both of these with the None default?

Reply:
Good point! Fixed!